client/jetpack-cloud/sections/partner-portal/stored-credit-card/index.tsx

```diff
@@ -0,0 +1,46 @@
+/**
+ * External dependencies
+ */
+import React, { ReactElement } from 'react';
+import { useTranslate } from 'i18n-calypso';
+
+/**
+ * Internal dependencies
+ */
+import { PaymentLogo } from '@automattic/composite-checkout';
+
+/**
+ * Style dependencies
+ */
+import './style.scss';
+
+export default function StoredCreditCard( props ): ReactElement {
+	const translate = useTranslate();
+	const creditCard = props.card;
+
+	const expiry = new Date( creditCard.expiry );
+	const month = expiry.getMonth();
+	const year = expiry.getFullYear().toString().substr( 2, 2 );
```

Comment:
This might be outside the scope but shouldn't we indicate if a card has expired in the card overview/list?

Reply:
Nice catch. I'll reach out to the design team.